crates/nu-cli/src/commands/char_.rs

```diff
@@ -64,6 +64,19 @@ fn str_to_character(s: &str) -> Option<String> {
     match s {
         "newline" | "enter" | "nl" => Some("\n".into()),
         "tab" => Some("\t".into()),
+        "branch" => Some('\u{e0a0}'.to_string().into()), // 
+        "branch_identical" => Some('\u{2263}'.to_string().into()), // ≣
+        "branch_untracked" => Some('\u{2262}'.to_string().into()), // ≢
+        "branch_ahead" => Some('\u{2191}'.to_string().into()), // ↑
+        "branch_behind" => Some('\u{2193}'.to_string().into()), // ↓
+        "branch_ahead_behind" => Some('\u{2195}'.to_string().into()), // ↕
+        "prompt" => Some('\u{25b6}'.to_string().into()), // ▶
+        "failed" => Some('\u{2a2f}'.to_string().into()), // ⨯
+        "elevated" => Some('\u{26a1}'.to_string().into()), // ⚡
+        "segment" => Some('\u{e0b0}'.to_string().into()), // 
+        "home" => Some("~".into()),                      // ~
+        "root" => Some("#".into()),                      // #
+        "hamburger" => Some('\u{2261}'.to_string().into()), // ≡
```

Comment:
Can we use the utf-8 names? Or are they unreadable?

Reply:
Maybe. Do you know how to figure that out? I do not.

Reply:
Most of these were taken from `oh-my-posh` and `posh-git` and in powershell these characters are enabled like this: `[char]0xe0a0`. Does every UTF-8 character have a name? I'm not sure where that list is. Google doesn't immediately know.

Reply:
Some of these seem to.

* https://www.compart.com/en/unicode/U+2193 (downward arrow)
* https://www.compart.com/en/unicode/U+2261(identical to???! hamburger seems better here)

Reply:
We chatted a bit more - sounds like it makes sense to have a few names for some, as people with different backgrounds will be looking for different things.

Reply:
Added synonyms. Please take a look.

Reply:
Looks good

Reply:
ok, landing then.